persistence/src/commonMain/db_user/com/wire/kalium/persistence/Users.sq

```diff
@@ -85,12 +85,12 @@ SELECT * FROM User
 WHERE (name LIKE ('%' || :searchQuery || '%')
 OR  handle LIKE  ('%' || :searchQuery || '%')
 OR  email LIKE  ('%' || :searchQuery || '%'))
-AND connection_status = :connectionStatus;
+AND connection_status IN :connectionStatuses;
```

Comment:
Where is this new `connectionStatuses` being declared?

Reply:
Shouldn't this need to be named `States` instead of `Statuses` ?

Reply:
i just added `es` to the end, but good point, changing it :)

Reply:
and about declaration: there is no need to declare it, we can just pass there `?` (the same as `: searchQuery` above) and SQLDelight "parse" it as a "second param in method", but to add more context to it me name this parameter.